isofit/radiative_transfer/luts.py

```diff
@@ -76,7 +80,7 @@ def initialize(
 
     ## Insert point dimensional keys
     # Filler arrays
-    dims = tuple(ds.dims.values())
+    dims = tuple(ds.sizes.values())
```

Comment:
`ds.dims.values()` triggers a `xarray` deprecation warning. This is the new way to achieve the same result.